test/liquid-drop-liquid-literal/index.spec.ts

```diff
@@ -0,0 +1,6 @@
+import { assertFormattedEqualsFixed } from '../test-helpers';
+import * as path from 'path';
+
+describe(`Unit: ${path.basename(__dirname)}`, () => {
```

Comment:
You know... that's not a bad idea :D I initially had all those because I didn't support `focus`, `debug` and `skip` in the liquid test definitions... But I eventually added them. Now I'm not sure we need this duplication anymore. We could definitely do a glob + run instead. 

I'll add that in a different PR.